pkg/fleet-manager/application/helper.go

```diff
@@ -60,6 +61,19 @@ func (a *ApplicationManager) syncPolicyResource(ctx context.Context, app *applic
 		}
 	}
 
+	// after finish instaill application, start handle rollout policy
+	rolloutCluster, err := a.fetchRolloutClusters(ctx, app, a.Client, fleet, syncPolicy)
```

Comment:
should this be under `if syncPolicy.Rollout != nil {` if it is not shared with other components